packages/avatar/src/vaadin-lit-avatar.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright (c) 2020 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import './vaadin-avatar-icons.js';
+import { html, LitElement } from 'lit';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { AvatarMixin } from './vaadin-avatar-mixin.js';
+import { avatarStyles } from './vaadin-avatar-styles.js';
+
+/**
+ * LitElement based version of `<vaadin-avatar>` web component.
+ *
+ * ## Disclaimer
+ *
+ * This component is an experiment not intended for publishing to npm.
+ * There is no ETA regarding specific Vaadin version where it'll land.
+ * Feel free to try this code in your apps as per Apache 2.0 license.
+ */
+class Avatar extends AvatarMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElement)))) {
+  static get is() {
+    return 'vaadin-avatar';
+  }
+
+  static get styles() {
+    return avatarStyles;
+  }
+
+  /** @protected */
+  ready() {
+    super.ready();
+
+    this._tooltipController = new TooltipController(this);
+    this.addController(this._tooltipController);
+  }
+
+  render() {
```

Comment:
Let's add `/** @protected */` JSDoc annotation to `render()` method.

Reply:
Done